src/components/Filters/CollapsibleFilter/CollapsibleFilter.js

```diff
@@ -3,8 +3,14 @@ import getIcon from '../../iconMap';
 import PropTypes from 'prop-types';
 import { useState } from 'react';
 
-const CollapsibleFilter = ({ className = '', title, desc, children }) => {
-  const [isOpen, setOpen] = useState(true);
+const CollapsibleFilter = ({
+  hasChildren = true,
+  className = '',
+  title,
+  desc,
+  children,
+}) => {
+  const [isOpen, setOpen] = useState(hasChildren);
```

Comment:
I thought we got rid of hasChildren, and it should just by default be:

const [isOpen, setOpen] = useState(true);

Reply:
We did, but it likely made its way back in due to a merge conflict on my end. I've taken these references out again.